tests/sentry/api/test_event_search.py

```diff
@@ -2439,7 +2439,7 @@ def test_count_at_least_function(self):
 
     def test_percentile_range(self):
         fields = [
-            "percentile_range(transaction.duration, 0.5, 2020-05-01T01:12:34, 2020-05-03T06:48:57, percentile_range_1)"
+            "percentile_range(transaction.duration, 0.5, 2020-05-01T01:12:34, 2020-05-03T06:48:57) as percentile_range_1"
```

Comment:
Do you see users using aliases in their requests?

Reply:
I'm hoping eventually they can, I'm thinking eventually it could be used so that `percentage` could easily be applied on other columns without having to know how we're constructing aliases under the hood.